### PR TITLE
Heat UI improvements

### PR DESCRIPTION
**Pull request by fdansv**
Adds:
* Trails 
* Fixes resolution for each mode
* Fixes form changed event not being called

@javisantana

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1428/))

File: lib/assets/javascripts/cartodb/table/menu_modules/wizards/torque_heat_wizard.js [hunk 1]
```diff
@@ -18,19 +18,21 @@ cdb.admin.mod.TorqueHeatWizard = cdb.admin.mod.TorqueWizard.extend({
...
         this.cartoProperties.set({
-          'torque-frame-count': 512
+          'torque-frame-count': 512,
+          'torque-resolution': 10 
         });
       }
       else{
         this.cartoProperties.set({
-          'torque-frame-count': 32
+          'torque-frame-count': 8
```
Comment (javisantana) [inline lib/assets/javascripts/cartodb/table/menu_modules/wizards/torque_heat_wizard.js]: 8 are too many, right?

Comment (fdansv) [inline lib/assets/javascripts/cartodb/table/menu_modules/wizards/torque_heat_wizard.js]: Oh, I accidentally changed that thinking it was resolution.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/1434/))

**Pull request merged**